site/src/atomics/border.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Border Atomics
+description: Border related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Border Atomics
+
+Various classes can be used to apply/customize borders on elements.
+
+- `.border`: sets a 1px border on an element
+- `.border-none`: resets border to none
+- `.border-{direction}`: sets a 1px border for that user direction
+- `.border-{direction}-none`: resets border for that user direction
```

Comment:
Perhaps add "to none" at the end to make things super clear? Same with line 16.

Reply:
Updated